app/code/Magento/Inventory/Model/SourceRepository.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model;
+
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\Framework\Exception\CouldNotSaveException;
+use Magento\Framework\Api\SearchCriteria\CollectionProcessorInterface;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\Model\AbstractModel;
+use Magento\InventoryApi\Api\Data\SourceInterface;
+use Magento\InventoryApi\Api\Data\SourceSearchResultsInterface;
+use Magento\InventoryApi\Api\SourceRepositoryInterface;
+use Magento\Inventory\Model\SourceSearchResultsFactory;
+use Magento\Inventory\Model\Resource\Source as ResourceSource;
+use Magento\Inventory\Model\Resource\Source\CollectionFactory;
+use Magento\Inventory\Model\SourceFactory;
+
+/**
+ * Class SourceRepository
+ * @package Magento\Inventory\Model
+ */
+class SourceRepository implements SourceRepositoryInterface
+{
+    /**
+     * @var ResourceSource
+     */
+    private $resource;
+
+    /**
+     * @var SourceFactory
+     */
+    private $sourceFactory;
+
+    /**
+     * @var CollectionProcessorInterface
+     */
+    private $collectionProcessor;
+
+    /**
+     * @var CollectionFactory
+     */
+    private $collectionFactory;
+
+    /**
+     * @var SourceSearchResultsFactory
+     */
+    private $sourceSearchResultsFactory;
+
+    /**
+     * SourceRepository constructor.
```

Comment:
we usually make a one line break between 
class description and parameter list

Also please provide meaningful description